.github/workflows/build.yml

```diff
@@ -22,41 +32,36 @@ jobs:
           fetch-depth: 0
       - name: Build binaries
         run: |
-          sudo apt-get update && sudo apt-get install -y osslsigncode
```

Comment:
Moved under the conditional so it's installed only when `matrix.os` is `windows`